### PR TITLE
cli: add stop command

### PR DESCRIPTION
**Pull request by okraskaj**
Signed-off-by: Jan Okraska <jan.okraska@cern.ch>

Connects #194.

### CODE REVIEW

File: docs/userguide.rst [hunk 1]
```diff
@@ -196,6 +196,25 @@ pass the --include-records flag:
...
+Passing no argument will stop the workflow selected by your REANA_WORKON
+variable. To specify a different workflow than your
+currently selected one use the -w/--workflow flag and set the workflow name
+or UUID.
+
+.. code-block:: console
+
+    $ reana-client stop --workflow=otherworkflow.1
+
+
```
Comment (dinosk) [inline docs/userguide.rst]: We need to mention here the `--force` flag

File: reana_client/cli/workflow.py [hunk 2]
```diff
@@ -513,6 +513,53 @@ def workflow_validate(ctx, file):
...
+    callback=workflow_uuid_or_name,
+    help='Name and run number to be stopped. '
+         'Overrides value of REANA_WORKON.')
+@add_access_token_options
+@click.pass_context
+@with_api_client
+def workflow_stop(ctx, workflow, force_stop, access_token):
+    """Stop given workflow."""
+    if not force_stop:
+        click.secho('Graceful stop not implement yet. If you really want to '
```
Comment (dinosk) [inline reana_client/cli/workflow.py]: Works fine like this, we could consider showing a confirmation message and having `--force` set by default

File: docs/userguide.rst [hunk 3]
```diff
@@ -196,6 +196,26 @@ pass the --include-records flag:
...
+Stopping workflows
+~~~~~~~~~~~~~~~~~~
+
+You can stop a workflow with:
+
+.. code-block:: console
+
+    $ reana-client stop --force
+
+Passing no argument will stop the workflow selected by your REANA_WORKON
```
Comment (roksys) [inline docs/userguide.rst]: I would rephrase this sentence to something like - 

> Workflow assigned to REANA_WORKON variable will be stoped. 

What do you think?
